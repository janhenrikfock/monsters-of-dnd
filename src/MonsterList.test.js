import React from 'react'
import { render } from '@testing-library/react'
import renderer from 'react-test-renderer'

import MonsterList from './MonsterList'

describe('MonsterList', () => {
  const headline = 'Monsters of DnD'
  const monsterListItems = 'Aboleth'

  it('displays the headline, followed by the first listitem which is aboleth', () => {
    const { getByText } = render(
      <MonsterList headline={headline} listitems={monsterListItems} />
    )

    expect(getByText(headline)).toBeInTheDocument()
    expect(monsterListItems).toMatchSnapshot()
  })
  it('renders correctly', () => {
    const list = renderer.create(
      <MonsterList headline={headline} listitem={monsterListItems} />
    )
    expect(list).toMatchSnapshot()
  })
})
