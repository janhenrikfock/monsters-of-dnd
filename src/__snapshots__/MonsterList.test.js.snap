// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MonsterList displays the headline, followed by the first listitem which is aboleth 1`] = `"Aboleth"`;

exports[`MonsterList renders correctly 1`] = `
<h1>
  Monsters of DnD
</h1>
`;
